node/package.json

```diff
@@ -38,6 +38,6 @@
     "merge2": "^0.3.6",
     "mocha": "^2.2.5",
     "semver": "^5.1.0",
-    "typescript": "next"
+    "typescript": "1.8.0-dev.20160104"
```

Comment:
This is a developer tools dependency only. This pins the CI server to a known tsc version that won't break our build.